pcsx2-qt/Settings/SettingsDialog.cpp

```diff
@@ -115,7 +115,7 @@ void SettingsDialog::setupUi(const GameList::Entry* game)
 	{
 		addWidget(m_game_fix_settings_widget = new GameFixSettingsWidget(this, m_ui.settingsContainer), tr("Game Fix"),
 			QStringLiteral("close-line"),
-			tr("<strong>Game Fix Settings</strong><hr>Gamefixes can work around incorrect emulation in some titles<br>however they can "
+			tr("<strong>Game Fix Settings</strong><hr>Gamefixes can work around incorrect emulation in some titles.<br>However, they can "
```

Comment:
```suggestion
			tr("<strong>Game Fix Settings</strong><hr>Game fixes can work around incorrect emulation in some titles.<br>However, they can "
```

Then maybe also this? Or vice versa? There are both "gamefix" and "game fix" on a single line. More consistensy wouldn't hurt.